bin/smack.py

```diff
@@ -233,6 +239,11 @@ def clang_frontend(args):
   smack_headers = os.path.join(smack_root, 'share', 'smack', 'include')
   smack_lib = os.path.join(smack_root, 'share', 'smack', 'lib', 'smack.c')
   smack_bc = temporary_file('smack', '.bc', args)
+  if args.pthread:
```

Comment:
These things also need to happen in the svcomp frontend, however, I wonder if we should break this out into a function, rather than copy/pasting this.